lib/logger/index.ts

```diff
@@ -6,13 +6,17 @@ import configSerializer from './config-serializer';
 import errSerializer from './err-serializer';
 import { RenovateStream } from './pretty-stdout';
 import type { BunyanRecord, Logger } from './types';
-import { ProblemStream, withSanitizer } from './utils';
+import { ProblemStream, isValidLogLevel, withSanitizer } from './utils';
 
 let logContext: string = process.env.LOG_CONTEXT ?? nanoid();
 let curMeta: Record<string, unknown> = {};
 
 const problems = new ProblemStream();
 
+process.env.LOG_LEVEL.toLocaleLowerCase();
+if (!isValidLogLevel(process.env.LOG_LEVEL as bunyan.LogLevel)) {
+  process.exit(1);
```

Comment:
We need to fatally log before exiting instead of exiting silently like this. I suggest that you should create the logger (skipping LOG_LEVEL) and then log a fatal error and exit after that.